score-server/src/main/java/bio/overture/score/server/util/Access.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2016 The Ontario Institute for Cancer Research. All rights reserved.
```

Comment:
Brand new files can provide a copyright with current year.

Reply:
With the latest PR changes have updated the new files with Copyright Notice